.github/workflows/swift-toolchain.yml

```diff
@@ -3874,26 +3878,26 @@ jobs:
         with:
           name: Windows-${{ matrix.arch }}-bld-msi
           path: |
-            ${{ github.workspace }}/BinaryCache/installer/Release/${{ inputs.build_arch }}/bld.msi
-            ${{ github.workspace }}/BinaryCache/installer/Release/${{ inputs.build_arch }}/bld.cab
+            ${{ github.workspace }}/BinaryCache/installer/Release/${{ matrix.arch }}/bld.msi
+            ${{ github.workspace }}/BinaryCache/installer/Release/${{ matrix.arch }}/bld.cab
```

Comment:
Hmm, I had to change this to `inputs.build_arch`. Does this work?

Reply:
If I'm holding things correctly, `ProductArchitecture` sets the output directory under `Release`. I noticed the package Tools job was not actually uploading anything because it was looking in the wrong folder. This should be set to the same value we give to `ProductArchitecture`, which is `matrix.arch` for this job.

Let's wait until the try job is done running, we'll know for sure.